### ISSUE TITLE
figDesc should be alt text only

### ISSUE THREAD

**Issue by ghost** [kirschbombe/mapsandtexts#6]
The `<figDesc>` element should be used only for image alt-text, e.g.:

https://github.com/kirschbombe/literaryparis/blob/master/articles/Asni%C3%A8res-sur-Seine.xml#L48

It is currently used in an image's popover.

**Comment by kirschbombe** [comment]
Use `<figDesc>` element in TEI docs to populate the HTML `alt=""` attribute with `<image>` for alt text?

**Comment by ghost** [comment]
Fixed with commit a6005d990a116a47a2cf53d4745d40661aa21d18
